Integrations/integration-ESM-v10.yml

```diff
@@ -302,17 +506,182 @@ script:
+        fixed_statuses = []
+        for status in case_statuses:
+            fixed_statuses.append({
+                'ID' : status['id'],
+                'Name' : status['name'],
+                'Is Default' : status['default'],
+                'Show In Case Pane' : status['showInCasePane']
+            })
+
+
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': fixed_statuses,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tblToMd('Result:', fixed_statuses, headers),
+            'EntryContext': {}
+        }
+
+    def organizations_to_entry(organizations):
+        if not organizations:
+            return 'No organizations were found'
+
+        headers = ['ID', 'Name']
+        fixed_organizations = []
+        for organization in organizations:
+            fixed_organizations.append({
+                'ID' : organization['id'],
+                'Name' : organization['name'],
+            })
+
+
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': fixed_organizations,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tblToMd('Case Events:', fixed_organizations, headers),
+            'EntryContext': {}
+       }
+
+    def events_to_entry(events):
+        if not events:
+            return 'No events were found'
+
+        headers = ['ID', 'LastTime', 'Message']
+        fixed_events = []
+        for event in events:
+            fixed_events.append({
+                'ID' : event['id']['value'].replace('|', '\|'),
+                'LastTime' : event['lastTime'],
+                'Message' : event['message'],
+            })
+
+        context = {'CaseEvents(val.ID && val.ID == obj.ID)' : fixed_events}
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': fixed_events,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tblToMd('Case Events:', fixed_events, headers),
+            'EntryContext': context
+       }
+
+    def users_to_entry(users):
+        # unreachable code - in order to send command, one must be logged in. therefore there is at least one user.
+        if not users:
+            return 'No users were found'
+
+        headers = ['ID', 'Name', 'Email', 'SMS', 'Master', 'Admin', 'Alias', ]
+        fixed_users = []
+        for user in users:
+            fixed_users.append({
+                'ID' : user['id']['value'],
+                'Name' : user['username'],
+                'Email' : user['email'],
+                'SMS' : user['sms'],
+                'Is Master' : user['master'],
+                'Is Admin' : user['admin'],
+            })
+
+        context = {'User(val.ID && val.ID == obj.ID)' : fixed_users}
```

Comment:
I think it should be `EsmUser` instead `User`
Plus - don't forget to update command output